src/System.Drawing.Common/dir.props

```diff
@@ -4,5 +4,6 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
Given this has a netfx asset we also need to bump the patch for the assembly version.

Reply:
Oh ok, thanks.